products/graalvm.md

```diff
@@ -17,7 +17,22 @@ releaseDateColumn: true
 #    # https://rubular.com/r/mQMD3DVwhwZf6M
 #    regex: '^vm-(?<major>[0-9]+)\.(?<minor>[0-9]+)\.(?<patch>[0-9]+)(\.(?<tiny>[0-9]+))?$'
 
+# See https://www.graalvm.org/release-calendar/
 releases:
+-   releaseCycle: "jdk-20"
+    releaseLabel: "JDK 20"
```

Comment:
Should we use the full name here to mark them distinctly separate? I'm not sure if `JDK X` is distinct enough.

Reply:
That's just my opinion, but it looks OK to me that way, and that's more consistent with what we have done on other products.